src/main/java/core/basesyntax/impl/ReportGeneratorServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.db.FruitShopStorage;
+import core.basesyntax.service.ReportGeneratorService;
+
+public class ReportGeneratorServiceImpl implements ReportGeneratorService {
+    private static final String SEPARATOR_COMMA = ",";
+    private static final String SEPARATOR_LINE = System.lineSeparator();
+    private static final String firstStringOfTheReport = "fruit,quantity";
+
+    @Override
+    public String generateReport(FruitShopStorage fruitShopStorage) {
+        StringBuilder stringBuilder = new StringBuilder();
```

Comment:
do not use such naming, how to guess what it stores?
```suggestion
        StringBuilder report = new StringBuilder();
```